dynamic-simulation/dynamic-simulation-tool/src/test/java/com/powsybl/dynamicsimulation/tool/DynamicSimulationToolTest.java

```diff
@@ -74,6 +79,52 @@ void testDynamicSimulation() throws IOException {
         assertCommand(new String[]{"dynamic-simulation", "--case-file", "/network.xiidm", "--dynamic-models-file", "/dynamicModels.groovy", "--curves-file", "/curves.groovy"}, 0, expectedOut, "");
     }
 
+    @Test
+    void testDynamicSimulationWithOutputFile() throws IOException {
+        String expectedOut = String.join(System.lineSeparator(),
+                "Loading network '/network.xiidm'",
+                "+ Dynamic Simulation Tool",
+                "Writing results to 'outputTest'" + System.lineSeparator());
+        String expectedOutputFile = """
+                        {
+                          "version" : "1.0",
+                          "isOK" : true,
+                          "logs" : null,
+                          "curves" : [ ],
+                          "timeLine" : {
+                            "metadata" : {
+                              "name" : "timeLine",
+                              "dataType" : "STRING",
+                              "tags" : [ ],
+                              "regularIndex" : {
+                                "startTime" : 0,
+                                "endTime" : 0,
+                                "spacing" : 0
+                              }
+                            },
+                            "chunks" : [ ]
+                          }
+                        }""";
+        assertCommand(new String[]{"dynamic-simulation", "--case-file", "/network.xiidm", "--dynamic-models-file", "/dynamicModels.groovy", "--output-file", "outputTest"}, 0, expectedOut, "");
+        ComparisonUtils.compareTxt(Files.newInputStream(fileSystem.getPath("outputTest")), expectedOutputFile);
```

Comment:
you should flip the arguments, as the first should be what is expected, and the second the actual text.
Could you also add a file extension in the name file?

Same applies for the next test